babel.config.js

```diff
@@ -0,0 +1,10 @@
+const { workspaces = [] } = require('./package.json');
+
+// This allows repo Jest running to find the individual .babelrc
```

Comment:
I don’t quite understand this line; should it be `repo running Jest`?

Reply:
(super-helpful comments tho!)

Reply:
Re-wrote the comment, should be clearer now.